x-pack/dockerlogbeat/pipelinemanager/config.go

```diff
@@ -0,0 +1,114 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package pipelinemanager
+
+import (
+	"crypto/sha1"
+	"errors"
+	"reflect"
+	"sort"
+)
+
+// ContainerOutputConfig has all the options we'll expect from --log-opts
+type ContainerOutputConfig struct {
+	Endpoint        string `config:"output.elasticsearch.hosts"`
```

Comment:
Do we want to support one host only? When working with the docker config file one might want to enter multiple hosts.

Reply:
This can't be trivially done, as docker's CLI will squash any duplicate config keys, and we get every config value as a string, We'll either need to tell users to do something like `endpoint1=..., endpoint2=...` or add the ability to parse lists from strings. Ideas?

Reply:
> add the ability to parse lists from strings

Yep, that's the idea. Just split on `,`.